src/content/ja/updates/2018/01/lighthouse.md

```diff
@@ -0,0 +1,94 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: New SEO audits and manual accessibility audits, and updates to the WebP audit.
+<span lang="ja-x-mtfrom-en">
+
+{# wf_updated_on: 2018-03-05 #}
+{# wf_published_on: 2018-01-05 #}
+{# wf_tags: lighthouse,accessibility,images #}
+{# wf_featured_image: /web/progressive-web-apps/images/pwa-lighthouse.png #}
+{# wf_featured_snippet: New SEO audits and manual accessibility audits, and updates to the WebP audit. #}
+{# wf_blink_components: N/A #}
+
+{% include "web/tools/chrome-devtools/_shared/styles.html" %}
+
+# ライトハウス2.7更 {: .page-title }
+
+{% include "web/_shared/contributors/vinamratasingal.html" %}
+{% include "web/_shared/contributors/patrickhulce.html" %}
+{% include "web/_shared/contributors/kaycebasques.html" %}
+
+[CDT]: /web/tools/lighthouse/#devtools
+[Node]: https://github.com/GoogleChrome/lighthouse#using-programmatically
+[CLI]: /web/tools/lighthouse/#cli
+[CE]: /web/tools/lighthouse/#extension
+
+Lighthouse 2.7が出ました！ハイライトは次のとおりです:
+
+* [新しいSEO監査](#seo)。
+* [新規のマニュアルアクセシビリティ監査](#a11y)。
+* [WebP監査の更新](#webp)。
+
+新機能、変更、バグ修正の完全なリストについては、[2.7リリースノート][RN]を参照してください。
+
+[RN]: https://github.com/GoogleChrome/lighthouse/releases/tag/v2.7.0
+
+## 2.7に更新する方法 {: #update }
+
+* NPM。 Lighthouseをグローバルにインストールした場合は、__ SPECIALWORD0または__ SPECIALWORD1フラグを実行します。
```

Comment:
What's this `SPECIALWORD` this?

Reply:
Ah, it looks like Translate is translating my __MARKER to __[space]MARKER  .. it seems to do this mostly for japanese.